Source/LightboxController.swift

```diff
@@ -303,10 +303,10 @@ open class LightboxController: UIViewController {
   // MARK: - Layout
 
   open func configureLayout(_ size: CGSize) {
-    scrollView.frame.size = size
+    scrollView.frame = calcScrollViewFrame(size)
     scrollView.contentSize = CGSize(
       width: size.width * CGFloat(numberOfPages) + spacing * CGFloat(numberOfPages - 1),
-      height: size.height)
+      height: scrollView.frame.height)
```

Comment:
This doesn't seem right...why are we setting the contentSize equal to the height of the frame? Doesn't that mean it can't scroll or pan?